docs/publish.md

```diff
@@ -1,29 +1,32 @@
 # Publish your app
 
+This page describes how to setup your CI/CD pipeline to upload your app to App Store Connect and Play Console.
```

Comment:
change to: Learn how to setup your CI/CD pipeline to upload your app to App Store Connect and Play Console.